.github/workflows/api-docs.yml

```diff
@@ -44,6 +53,12 @@ jobs:
           python3 scripts/gen_vimdoc.py
           printf '::set-output name=UPDATED_DOCS::%s\n' $([ -z "$(git diff)" ]; echo $?)
 
+      - name: FAIL, PR has not commited doc changes
+        if: ${{ steps.docs.outputs.UPDATED_DOCS != 0 && inputs.check_only }}
+        run: |
+          echo "Job failed, run ./scripts/gen_vimdoc.py and commit your doc changes"
+          exit 1
+
       - name: Automatic PR
         if: ${{ steps.docs.outputs.UPDATED_DOCS != 0 }}
```

Comment:
Also there should be a `!inputs.check_only` here

Reply:
Yeah this must be it, `check_only` is false and `UPDATE_DOCS != 0`, the only way it reaches here from a workflow_call, but the weird thing is the only workflow call there is comes from [here](https://github.com/muniter/neovim/blob/91914010252a1f5c8a3358d09c3be92f30585ee3/.github/workflows/api-docs-check.yml#L17) where we pass `check_only` true.

@zeertzjq Did you get this error from a rerun of the job perhaps?

cc @dundargoc

Reply:
Hmm I cannot understand what you mean.

Reply:
The error is unrelated to this step. The whole workflow fails to be run.

Reply:
Oh now I understand, thanks.